### ISSUE TITLE
Select elements with v-model not working (Vue 3)

### ISSUE THREAD

**Issue by anatolsommer** [vitest-dev/vitest#1013]
### Describe the bug

Using select elements with v-model in a Vue 3 SFC throws an error when mounting the component using vue-test-utils. This might be a problem with happy-dom since it seems to work with jsdom or a bug in vue-test-utils or vue so I'm not sure where this issue belongs.

### Reproduction

https://github.com/anatolsommer/vue3-vitest-select-issue
Run `npm test`
(Error message also included in readme file)

### System Info

```shell
System:
    OS: Linux 5.4 Ubuntu 20.04.1 LTS (Focal Fossa)
    CPU: (6) x64 Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz
    Memory: 1.48 GB / 3.76 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 16.14.0 - /usr/local/bin/node
    npm: 8.3.1 - /usr/local/bin/npm
  npmPackages:
    @vitejs/plugin-vue: ^2.2.0 => 2.2.2 
    vite: ^2.8.0 => 2.8.4 
    vitest: ^0.6.0 => 0.6.0 
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sheremet-va** [comment]
This a `happy-dom` issue. Please create an issue in their repo. `<SELECT>` element doesn't have `options` property for some reason.

**Comment by anatolsommer** [comment]
Fixed in happy-dom v5.1.0
